mysql-plugin/src/test/java/io/cdap/plugin/mysql/MysqlSinkTestRun.java

```diff
@@ -40,9 +40,12 @@
 import java.sql.Timestamp;
 import java.time.LocalDateTime;
 import java.util.ArrayList;
-import java.util.Arrays;
 import java.util.List;
 
+import static io.cdap.plugin.db.CustomAssertions.assertBytesEquals;
```

Comment:
style nit: we don't use static method imports so that it is more clear when a static call is being made. Method calls should be changed to CustomAssertions.assertBytesEquals().

Reply:
Thanks. Changed.